### ISSUE TITLE
Accessibility issue with using Swing or AWT as our GUI platform

### ISSUE THREAD

**Issue by DandyEnders** [DandyEnders/Braille-Project#6]
# Problem
The initial intention of using Swing and AWT is to have a GUI tool for us to code look of windows in our program. But it was soon figured out that Swing and AWT are an old GUI APIs which do not support a new feature such as accessibility feature. ( **i.e. visually impaired users cannot use our program with their screen reader using Swing or AWT** ) This caused a massive problem in our program structure because everything We've built on was based on Swing. 

# Solution
**We have to convert Swing code to function in JavaFX because it supports accessibility features like screen reader supports**; JavaFX is another Java GUI API just like Swing and AWT but robust and easier to structure. Using JavaFX in our program, the problem above can be solved and structured easily into MVC ( model, view, controller ), making our program more maintainable.

# Progress
I have so far implemented Swing version of our GUI [MainMenu](https://github.com/DandyEnders/Braille-Project/tree/master/Enamel/src/mainMenuSwing), [ScenarioEditor](https://github.com/DandyEnders/Braille-Project/tree/master/Enamel/src/scenarioEditorMenuSwing) into JavaFX. No further functionality has implemented.
Commit when this Issue was created 2d6aacbf5c45535c4ee58eb78ff316ac2c54c71b

**Comment by DandyEnders** [comment]
# Progress
I implemented save button and refined load scenario by catching errors that are fired if a user cancels to load.
